.github/workflows/android-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: Unit tests
+name: Android Tests
```

Comment:
This name is misleading.
All of our Android tests run in firebase test lab service.
Tests in this action run in a Dart VM. The VM runs in Linux environment. There is nothing Android-related.

Reply:
By the way, I initially intended this action to also hold any other unit tests. For example, for our C++ code.